fsspec/spec.py

```diff
@@ -976,6 +976,12 @@ def download(self, rpath, lpath, recursive=False, **kwargs):
         """Alias of :ref:`FilesystemSpec.get`."""
         return self.get(rpath, lpath, recursive=recursive, **kwargs)
 
+    def created(self, path):
```

Comment:
They sound pretty self-explanatory, but would be good to include docstrings anyway.